### DISCUSSION TITLE
Clementine cask does not produce a working executable

### DISCUSSION THREAD

**Discussion by trg818** [Homebrew/discussions#1093] [Casks]
After having tried unsuccessfully to start my old [clementine](https://www.clementine-player.org/) music player on the new MacBook Pro under Big Sur, I went for the cask provided by Homebrew hoping for better results. The package did seemingly install correctly and without complaints, but when starting the player, it crashed immediately. Is this problem known?
I desisted from posting the lengthy crash report here right away; let me know if that would help.

**Comment by SMillerDev** [comment]
Casks just install the upstream installer. You should report it to the developers of it crashes.

**Reply by trg818** [reply]
I see. I have done so now, and it seems that others also had the problem already on Mojave. I also tried to compile from source now, following [these instructions](https://github.com/clementine-player/Clementine/wiki/Compiling-from-source-(macOS)), but it turns out that they themselves seem to be broken, and they didn't work on my machine.

It's a pity that Clementine seems to have been sliding into a sort of zombie state in the past years, given that there seems to be ongoing work on the code and even a v.1.4rc but no actual release since 2016, so I won't hold my breath waiting for it to be fixed.
